src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -55,9 +56,16 @@ public ReadPileup(final Locatable loc, final Map<String, ReadPileup> stratifiedP
      * Create a new pileup without any aligned reads
      */
     public ReadPileup(final Locatable loc) {
-        this(loc, Collections.emptyList());
+        this(loc, new ArrayList<>());
     }
 
+//    /**
```

Comment:
I'm fine with deleting this.

Reply:
more garbage I forgot to clean out before posting the PR